src/java.base/share/man/java.1

```diff
@@ -5533,6 +5533,37 @@ application start\-up script with the same technique:
 .PP
 \f[CB]env\ JAVA_TOOL_OPTIONS=\-XX:+RecordDynamicDumpInfo\ bash\ app_start.sh\f[R]
 .RE
+.SS Using Dynamic CDS Archive File with \-XX:+AutoCreateSharedArchive
+.PP
+\f[CB]\-XX:+AutoCreateSharedArchive\f[R] is a more convenient way of
+creating/using CDS archives.
+Unlike the methods of manual CDS archive creation described in the
+previous section, with \f[CB]\-XX:+AutoCreateSharedArchive\f[R], it\[aq]s
+no longer necessary to have a separate trial run.
+Instead, you can always run the application with the same command\-line
+and enjoy the benefits of CDS automatically.
+.RS
+.PP
+\f[CB]java\ \-XX:+AutoCreateSharedArchive\ \-XX:SharedArchiveFile=hello.jsa\ \-cp\ hello.jar\ Hello\f[R]
+.RE
+.IP \[bu] 2
+If the specified archive file exists and is created by the same version
+of the JDK, the JVM will load this file as a dynamic archive.
+.IP \[bu] 2
+Otherwise the JVM will not load the specified archive.
+Instead, it will automatically create (or update) the archive file at
+exit.
+As a result, the next time you run the same command again, the archive
+will be available and ready to be used.
+.PP
+The contents of the CDS archive file are specific to each version of the
+JDK.
+Therefore, if you upgrade to a newer JDK,
+\f[CB]\-XX:+AutoCreateSharedArchive\f[R] will automatically recreate the
+archive.
```

Comment:
But the first time you run it won't be able to use the existing archive file. This also suggests that the answer above is that the file will always get written at exit, regardless of the version or whether it pre-existed.

Reply:
Hi David, I have updated the text to be more precise about the behavior.